ui/src/app/pages/data/cohort/cohort-criteria-menu.tsx

```diff
@@ -375,145 +387,138 @@ export const CohortCriteriaMenu = withCurrentWorkspace()(
                     <Spinner size={36} />
                   </div>
                 ) : (
-                  menuOptions
-                    .filter((optionList) =>
-                      isTemporal
-                        ? optionList[0].category !== 'Program Data'
-                        : optionList
-                    )
-                    .map(
-                      (category, index) =>
-                        showMenuItem(index) && (
-                          <ul key={index}>
-                            <li
-                              style={styles.dropdownHeader}
-                              className='menuitem-header'
-                            >
-                              <span style={styles.dropdownHeaderText}>
-                                {category[0].category}
-                              </span>
-                            </li>
-                            {category
-                              .filter(
-                                (menuItem) =>
-                                  domainCounts === null ||
-                                  domainCounts.find(
-                                    (dc) => dc.domain === menuItem.domain
-                                  )
-                              )
-                              .map((menuItem, m) => (
-                                <li
+                  (isTemporal ? temporalFilter(menuOptions) : menuOptions).map(
```

Comment:
FYI this is the last line changed in this file. Everything below is indenting/formatting, just looks like a lot of changes in Github